ansible/roles/validate/tasks/main.yaml

```diff
@@ -1,7 +1,7 @@
 ---
   - name: check for supported OS
     fail:
-      msg: "{{ ansible_distribution }} {{ ansible_distribution_major_version }} is not supported. The supported OSes - CentOS 7/8, RHEL 7/8, Suse 15, SLES 15, Ubuntu 16 (xenial) 18 (bionic) 20 (focal) or Debian 9 (stretch)"
+      msg: "{{ ansible_distribution }} {{ ansible_distribution_major_version }} is not supported. The supported OSes - CentOS 7/8, RHEL 7/8, Oracle 7/8, Suse 15, SLES 15, Ubuntu 16 (xenial) 18 (bionic) 20 (focal) or Debian 9 (stretch)"
```

Comment:
What is `ansible_distribution` for Oracle 7/8, is it already covered in [here](https://github.com/mesosphere/konvoy-image-builder/pull/119/files#diff-a1e869ad41686e5506dfee8e1e9e2c602ebc623d60d8ab23eceeaaa0a1c2175fR5-R14)?

Reply:
> What is `ansible_distribution` for Oracle 7/8, is it already covered in [here](https://github.com/mesosphere/konvoy-image-builder/pull/119/files#diff-a1e869ad41686e5506dfee8e1e9e2c602ebc623d60d8ab23eceeaaa0a1c2175fR5-R14)?

Yes - Oracle's Linux distribution is detected as "Red Hat Enterprise Linux Server". Should I remove "Oracle 7/8," from this fail messge?

Reply:
Had an offline discussion with @raemar 
Ansible sees Oracle 7/8 as:
```
(ansible_distribution == "RedHat" and ansible_distribution_major_version|int in [7, 8])
```

This explains how the rest of the plays just work.